clients/python/game/models.py

```diff
@@ -0,0 +1,18 @@
+class Game:
+    def __init__(self, id_, tick, players, resources, map_, viruses):
+        self.id = id_
+        self.tick = tick
+        self.players = players
+        self.resources = resources
+        self.map = map_
+        self.viruses = viruses
```

Comment:
I think that's just because we didn't create issues for everything yet. Since we originally planned to add them, I would prefer to leave it like that (so that we don't forget to add it to the python API). If we decide to remove them on #214 , I'll remove that line. How does that sound?